spike_main/spike.cc

```diff
@@ -241,6 +242,7 @@ int main(int argc, char** argv)
   uint16_t rbb_port = 0;
   bool use_rbb = false;
   unsigned dmi_rti = 0;
+  reg_t blocksz = 16;
```

Comment:
I agree with https://github.com/riscv/sail-riscv/pull/137#discussion_r771784779 and default block size would be better to be 64.

Reply:
OK, I'll fix this. There is still some problem unsolved in the review of sail-riscv, this may also affect the implementation of spike.